grpc.echo/zilla.yaml

```diff
@@ -1,4 +1,11 @@
 name: example
+catalogs:
+  catalog0:
```

Comment:
Can you update all of the catalog names to be `host_filesystem` or something more descriptive of what the catalog is doing.